includes/class-sensei-course.php

```diff
@@ -193,9 +201,108 @@ public function highlight_menu_item( $submenu_file ) {
+	 * @internal
+	 * @access private
+	 */
+	public function showcase_courses_screen() {
+		// Enqueue styles.
+		Sensei()->assets->enqueue( 'sensei-showcase-upsell', 'css/showcase-upsell.css' );
+
+		$illustration = Sensei()->assets->get_image( 'showcase-courses-upsell-illustration.png' );
+		?>
+		<div class="wrap">
+			<?php
+				$screen = get_current_screen();
+				$tabs   = $this->get_course_custom_navigation_tabs();
+
+				$this->display_courses_navigation( $screen, $tabs );
+			?>
+			<div class="sensei-showcase-upsell">
+				<div class="sensei-showcase-upsell__content">
+					<h1 class="sensei-showcase-upsell__title"><?php esc_html_e( 'Showcase your course with Sensei Pro!', 'sensei-lms' ); ?></h1>
+					<p class="sensei-showcase-upsell__description"><?php esc_html_e( 'Gain visibility by promoting your post on Sensei Showcase Gallery page.', 'sensei-lms' ); ?></p>
+
+					<ul class="sensei-showcase-upsell__list">
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'Free traffic redirected to your site', 'sensei-lms' ); ?></li>
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'WooCommerce integration', 'sensei-lms' ); ?></li>
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'Schedule ‘drip’ content', 'sensei-lms' ); ?></li>
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'Set expiration date of courses', 'sensei-lms' ); ?></li>
+						<li class="sensei-showcase-upsell__list-item dashicons-before"><?php esc_html_e( 'Interactive Blocks', 'sensei-lms' ); ?></li>
+					</ul>
+
+					<div class="sensei-showcase-upsell__price-wrapper">
+						<span class="sensei-showcase-upsell__price"><?php esc_html_e( '$13 USD', 'sensei-lms' ); ?></span>
+						<span class="sensei-showcase-upsell__price-period"><?php esc_html_e( 'per month, billed yearly', 'sensei-lms' ); ?></span>
+					</div>
+
+					<ul class="sensei-showcase-upsell__buttons">
+						<li><a href="https://senseilms.com/sensei-pro/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=showcase" class="sensei-showcase-upsell__button sensei-showcase-upsell__button--primary" target="_blank" rel="noreferrer"><?php esc_html_e( 'Get Sensei Pro', 'sensei-lms' ); ?></a></li>
+						<li><a href="#" class="sensei-showcase-upsell__button sensei-showcase-upsell__button--secondary" target="_blank" rel="noreferrer"><?php esc_html_e( 'Learn more', 'sensei-lms' ); ?></a></li>
+					</ul>
+				</div>
+
+				<div class="sensei-showcase-upsell__illustration-wrapper">
+					<img class="sensei-showcase-upsell__illustration" src="<?php echo esc_url( $illustration ); ?>" alt="<?php echo esc_attr( __( 'Illustration for Showcase Courses', 'sensei-lms' ) ); ?>" />
+				</div>
+			</div>
+		</div>
+		<?php
+	}
+
+	/**
+	 * Filter the showcase courses page title. It's needed because submenu pages
+	 * without a parent menu item don't have a title.
```

Comment:
I didn't understand this, doesn't this page have a parent menu? (the Sensei Courses one?)

Reply:
I think it would be ideal, to fix the title and the menu highlight without the filters, but it mess up with the menu highlight.
So I followed the "Order Courses" approach with the parent as `null`, and highlight it in the menu through this [conditional](https://github.com/Automattic/sensei/pull/6648/files#diff-a700db88caca1edaa7ce24fc214d9344458fb9a1a636e4f8fa6bfb9eaa9a93dfR219.). If you know any approach that would work, I'd appreciate it. :)